example/main.py

```diff
@@ -1,18 +1,20 @@
 import os
 from dotenv import load_dotenv
 from omnilog.send_to_db import send_to_db
-from omnilog.startListener import startListener
+from omnilog.start_listener import start_listener
 from llm.get_openai_response import get_openai_response
 
-startListener()
+start_listener()
 
 prompt = "What is a proompt?"
 log = get_openai_response(prompt)
-log["input"]= prompt
+log["input"] = prompt
 
 load_dotenv()
 DATABASE_URL = os.getenv("DATABASE_URL")
 if not DATABASE_URL:
-    raise EnvironmentError(f"DATABASE_URL not found: {DATABASE_URL}\nMake sure that you have a DATABASE_URL value in the .env file.")
```

Comment:
why does this keep changing every PR btw?

Reply:
I am not sure, but having the argument on a new line is to follow black formatting and is the correct version